src/main/java/edu/ie3/util/StreamUtils.java

```diff
@@ -18,6 +18,11 @@
  * @since 13.04.19
  */
 public class StreamUtils {
+
+  private StreamUtils() {
+    throw new IllegalStateException("Utility class");
```

Comment:
Please use a meaningful message. I would suggest `"This is a Utility Class and not meant to be instantiated."` from `FileIoUtils`. Might also be an idea to put this string into a `UtilityHelper`-class or so to have the same message in each utility class.

Reply:
ACK.